src/Core/SecretsManager/Entities/SMImport.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Bit.Core.SecretsManager.Entities;
+
+public class SMImport
```

Comment:
Why do we have an entity for this? Entities are traditionally stored in the database.

Reply:
Great point, fixed